tests/foreman/api/test_classparameters.py

```diff
@@ -416,15 +626,25 @@ def test_positive_validate_default_value_with_list(self):
         3. Validate this value with list validator type and rule.
 
         @assert: Error not raised for default value in list.
-
-        @caseautomation: notautomated
-
-        @CaseLevel: System
         """
+        sc_param = self.sc_params_list.pop()
+        sc_param.override = True
+        sc_param.default_value = 'test'
```

Comment:
My intention was to test the case with multiple values may be with multiple data types instead of testing it with only single data point.